src/tests/test_sample.py

```diff
@@ -5,6 +5,11 @@
 You can also group multiple tests in a class: https://pytest.org/en/7.4.x/getting-started.html#group-multiple-tests-in-a-class
 """
 
+from unittest import main  # , TestCase
 
-def test_sample():
-  assert 4 == 4
+# class SampleTestCase(TestCase):
+#   def test_sample(self):
+#     assert 4 == 4
+
+if __name__ == "__name__":
+  main()
```

Comment:
Not sure what `main()` is running. Might just be better to remove the tests.

Then again, that would re-introduce the annoying prompt every time we deploy. If this fix removes the prompt, probably better to keep it.

Reply:
Unittest.main will run the test classes we create, if we create them. The side issue is that the robotpy test hangs trying to run the same tests, but we can circumvent that by deploying with --skip-tests tag and just use standard python unittest for our workflow. But in fairness I don't have any bandwidth to introduce the students to unit testing. They would need to pick that up on their own, or someone else can give them a quick primer at some point.